questions.txt

```diff
@@ -0,0 +1,16 @@
+questions
+
+
```

Comment:
Great job on the self-study questions! if would be easier to answer the questions within the README.md file